integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -625,23 +625,19 @@ def test_min_max_group_by(data_gen):
             .agg(f.min('b'), f.max('b')))
 
 # To avoid ordering issues with collect_list, sorting the arrays that are returned.
-# Note, using sort_array() on the CPU, because sort_array() does not yet
+# NOTE: sorting the arrays locally, because sort_array() does not yet
 # support sorting certain nested/arbitrary types on the GPU
 # See https://github.com/NVIDIA/spark-rapids/issues/3715
 # and https://github.com/rapidsai/cudf/issues/11222
-@allow_non_gpu("ProjectExec", "SortArray", *non_utc_allow)
-@ignore_order(local=True)
+@allow_non_gpu("ProjectExec", *non_utc_allow)
+@ignore_order(local=True, arrays=["blist"])
```

Comment:
we do not have many instances of this marker so it looks fine. However, is there anything that is in the way of making the `arrays` parameter implicit.

for tests with this marker can we not just infer the value of `arrays` by looking at the columns with the type `ArrayType`?

Reply:
I had actually just thought about this possibility after pushing, will investigate. I wasn't entirely sure about making it implicit, because I wasn't certain about the possibility of array ordering mattering. I could definitely infer the schema though.

Reply:
It is fine to defer it to  a follow-up issue. Probably if we had a convention in test to prefix output columns it could be done quite safely after filtering by type.